.github/workflows/automated_release.yaml

```diff
@@ -0,0 +1,66 @@
+name: Automated release for PyAqueduct every alternate Monday at 12:00 OM
```

Comment:
The way ci status badges work, is that it generates a SVG out of action name, so what about keeping it short? but we have the full name in comment
```suggestion
name: Automated release
# Automated release for Aqueduct Core every alternate Monday at 12:00 PM
```